<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>420</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="9" column="2" colspan="3">
     <widget class="QLabel" name="pixmapNameLabel">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="5" colspan="4">
     <widget class="QProgressBar" name="progress1Bar">
      <property name="toolTip">
       <string>Threadstatus</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="approximationLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0" rowspan="7" colspan="2">
     <widget class="QListWidget" name="listWidget">
      <property name="toolTip">
       <string>Identical Imagelist</string>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="identicalImageBox">
      <property name="toolTip">
       <string>The count of identical found images </string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Identical images count:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="timelabel">
      <property name="text">
       <string>Time lapse:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="5" colspan="4">
     <widget class="QLabel" name="identicalPixmapNameLabel">
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2" rowspan="6" colspan="3">
     <widget class="QLabel" name="pixmapLabel">
      <property name="minimumSize">
       <size>
        <width>229</width>
        <height>183</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>229</width>
        <height>183</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Shows the selected image</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="5" colspan="2">
     <widget class="QPushButton" name="abortButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Cancel the current process</string>
      </property>
      <property name="text">
       <string>&amp;abort</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QPushButton" name="startButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Start to compare images</string>
      </property>
      <property name="text">
       <string>&amp;start</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="4">
     <widget class="QProgressBar" name="progressBar">
      <property name="toolTip">
       <string>Processstatus</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="10" column="7" colspan="2">
     <widget class="QPushButton" name="closeButton">
      <property name="toolTip">
       <string>Close this application</string>
      </property>
      <property name="text">
       <string>&amp;close</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5" colspan="4">
     <widget class="QProgressBar" name="progress3Bar">
      <property name="toolTip">
       <string>Threadstatus</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="6" colspan="3">
     <widget class="QTimeEdit" name="timeEdit">
      <property name="toolTip">
       <string>Shows the lapes time after the process finisched</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="displayFormat">
       <string>HH:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5" colspan="2">
     <widget class="QPushButton" name="deleteButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Remove the selected Image</string>
      </property>
      <property name="text">
       <string>&amp;delete</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5" colspan="4">
     <widget class="QProgressBar" name="progress2Bar">
      <property name="toolTip">
       <string>Threadstatus</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QSlider" name="approximationSlider">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Approximation settings:&lt;/p&gt;&lt;p&gt;Fast = Image has to be exact equal.&lt;/p&gt;&lt;p&gt;x% =  A&lt;span style=&quot; font-family:'Hack';&quot;&gt;pproach method the result could&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Hack';&quot;&gt;be different depends on the size of images&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Hack';&quot;&gt;which comparing.    &lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="maximum">
       <number>35</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="5" colspan="4">
     <widget class="QProgressBar" name="progress0Bar">
      <property name="toolTip">
       <string>Threadstatus</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="5" colspan="2">
     <widget class="QLabel" name="identicalwithlabel">
      <property name="text">
       <string>Identical with:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="9">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Path:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QCheckBox" name="allFormatsCheckBox">
         <property name="toolTip">
          <string>Read all supported image formats in the current path</string>
         </property>
         <property name="text">
          <string>All formats</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="formatBox">
         <property name="toolTip">
          <string>Shows the supported image formats e. g. png</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Format:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4" colspan="2">
        <widget class="QCheckBox" name="subdirectoriesCheckBox">
         <property name="toolTip">
          <string>Including the subfolders inside the directory</string>
         </property>
         <property name="text">
          <string>Include subdirectories</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QToolButton" name="pathButton">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Open a DirectorieDialog to choose a directorie</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QSpinBox" name="imageCountBox">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>The count of images</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="5">
        <widget class="QLineEdit" name="pathEdit">
         <property name="toolTip">
          <string>The current directory</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QSpinBox" name="coreBox">
         <property name="toolTip">
          <string>Count of threads to use for comparing images</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Threads:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="7" colspan="2">
     <widget class="QToolButton" name="infoButton">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>0</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>85</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>85</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>165</red>
            <green>167</green>
            <blue>169</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>172</red>
            <green>174</green>
            <blue>176</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>165</red>
            <green>167</green>
            <blue>169</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open an InfoDialog &lt;/p&gt;&lt;p&gt;Shows a little description how to use this application. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>i</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pathEdit</tabstop>
  <tabstop>formatBox</tabstop>
  <tabstop>allFormatsCheckBox</tabstop>
  <tabstop>subdirectoriesCheckBox</tabstop>
  <tabstop>pathButton</tabstop>
  <tabstop>coreBox</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>abortButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>timeEdit</tabstop>
  <tabstop>identicalImageBox</tabstop>
  <tabstop>imageCountBox</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
